Mapping/ClassMetadataInterface.php

```diff
@@ -77,7 +77,7 @@ public function getReflectionClass();
     /**
      * Gets the attribute identifier of the class.
      *
-     * @return AttributeMetadataInterface
+     * @return AttributeMetadataInterface|null
```

Comment:
I'm not comfortable with `null` result, which IMO is the same than mixed results. If we call the `getIdentifier` method, we expect the identifier to be there. If there's no identifier, then throw a `IdentifierNotFound` exception for instance.